terrahelp/mask.go

```diff
@@ -53,8 +55,11 @@ const (
 	ThMaskChar    = "*"
 	ThMaskCharNum = 6
 
-	ThPrev2CurrPattern        = "\"(.+)\"\\s*=>\\s*\"(\\%s*)\""
-	ThPrev2CurrReplacePattern = "\"%s\" => \"%s\""
+	ThPrev2CurrPatternPre012        = "\"(.+)\"\\s*=>\\s*\"(\\%s*)\""
```

Comment:
[Minor] I don understand this variable names, 
Th - > terr help
Prev2 is not pre hcl2 or review ?
Curr -> For current ? if tho that is no longer true, I will be happy to see something more meaningful.

Reply:
I have updated the variable names to the following 
`ThPrev2CurrPattern` became `PrevVal2CurrentValConsolePattern` 

Which is "previous value to current value console pattern".
The above naming has been also used for the Pre 0.12 values and they would read `PrevVal2CurrentValConsolePatternPre012`